lib/minesweeper_2pl/app.rb

```diff
@@ -1,7 +1,41 @@
-module Minesweeper_2pl
+module Minesweeper
   class App
-    def call
-      "Hello World"
+    BOMB_PERCENT = 0.75
```

Comment:
These constants (`BOMB_PERCENT` and `MAX_ROW_NUM`) might be more logically related to something else in your gem. `Game`? `Board`? Think about class cohesion...where do they belong?